themes/OpenFisca-2022/layouts/_default/packages.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="{{ $.Site.Language }}">
+
+    {{ partial "head.html" . }}
+
+    <body>
+        {{ partial "sprite.svg" . }}
+
+        {{ partial "header.html" . }}
```

Comment:
Please [inherit from baseof](https://gohugo.io/templates/base/#override-the-base-template) for maintainability :)